types/index.d.ts

```diff
@@ -9,13 +9,13 @@ export type {
   TransformedTokens,
```

Comment:
Let's add Dictionary as well here, pretty commonly used especially in format hook